python/lsst/meas/base/base.py

```diff
@@ -181,24 +206,28 @@ class BasePlugin(object):
     This is the base class for SingleFramePlugin and ForcedPlugin; derived classes should inherit
     from one of those.
     """
-
-    @staticmethod
-    def getExecutionOrder():
+    # named class constants for execution order
+    CENTROID_ORDER = 0.0
+    SHAPE_ORDER = 1.0
+    FLUX_ORDER = 2.0
+    APCORR_ORDER = 4.0
+    CLASSIFY_ORDER = 5.0
+
+    @classmethod
+    def getExecutionOrder(cls):
         """Sets the relative order of plugins (smaller numbers run first).
 
-        In general, the following values should be used (intermediate values
+        In general, the following class constants should be used (other values
         are also allowed, but should be avoided unless they are needed):
-        0.0 ------ centroids and other algorithms that require only a Footprint and
-                   its Peaks as input
-        1.0 ------ shape measurements and other algorithms that require
-                   getCentroid() to return a good centroid in addition to a
-                   Footprint and its Peaks.
-        2.0 ------ flux algorithms that require both getShape() and getCentroid()
-                   in addition to the Footprint and its Peaks
-        3.0 ------ algorithms that operate on fluxes (e.g. classification,
-                   aperture correction).
-
-        Must be reimplemented (as a static or class method) by concrete derived classes.
+        CENTROID_ORDER  centroids and other algorithms that require only a Footprint and its Peaks as input
+        SHAPE_ORDER     shape measurements and other algorithms that require getCentroid() to return
+                        a good centroid (in addition to a Footprint and its Peaks).
+        FLUX_ORDER      flux algorithms that require both getShape() and getCentroid(),
+                        in addition to a Footprint and its Peaks
+        APCORR_ORDER    algorithms that operate on fluxes (e.g. aperture correction)
+        CLASSIFY_ORDER  algorithms that operate on aperture-corrected fluxes
+
+        Must be reimplemented as a class method by concrete derived classes.
```

Comment:
Let's just make the documentation for `APCORR_ORDER` "the executionOrder at which aperture correction is run"; we don't want to encourage plugins to use this order, I think, as we don't want them to have to worry about what happens in case of a tie.